internal/network/command_dns_forwarder_create.go

```diff
@@ -0,0 +1,93 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingdnsforwarderv1 "github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-dnsforwarder/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newDnsForwarderCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create a DNS forwarder.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.dnsForwarderCreate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Create a DNS forwarder.",
+				Code: "confluent network dns forwarder create --domains abc.com,def.com --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456",
+			},
+			examples.Example{
+				Text: "Create a named DNS forwarder.",
+				Code: "confluent network dns forwarder my-dns-forwarder create --domains abc.com,def.com --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456",
+			},
+		),
+	}
+
+	cmd.Flags().StringSlice("dns-server-ips", nil, "A comma-separated list of IP addresses for the DNS server.")
+	cmd.Flags().String("gateway", "", "Gateway ID.")
```

Comment:
DNS gateway ID?

Reply:
Gateway is a new resource analogous to a network